docs/DeveloperGuide.md

```diff
@@ -1,5 +1,26 @@
 # Developer Guide
 
```

Comment:
Better have more examples to each command, users will not know how to invoke the method without method signature or example.